package.json

```diff
@@ -68,7 +68,8 @@
   "scripts": {
     "start": "vite",
     "build": "vite build",
-    "test": "cross-env CI=true vitest --coverage",
+    "test": "cross-env CI=true vitest dev",
```

Comment:
You changed `test` script, but this script is used in `.husky/pre-push`. Maybe you should only add new script without changing of existing?

Reply:
Returned the original test command and added a new test-dev command to test in dev mode.